packages/discord.js/src/util/ChannelFlagsBitField.js

```diff
@@ -0,0 +1,32 @@
+'use strict';
+
+const { ChannelFlags } = require('discord-api-types/v10');
+const BitField = require('./BitField');
+
+/**
+ * Data structure that makes it easy to interact with a {@link BaseChannel#flags} bitfield.
+ * @extends {BitField}
+ */
+class ChannelFlagsBitField extends BitField {
+  /**
+   * Numeric guild channel flags.
+   * @type {ChannelFlags}
```

Comment:
Don't think this type exists? Should probably be added to the [APITypes](https://github.com/discordjs/discord.js/blob/e3d16c6950d4e3e43013adb503107807d4f39ae8/packages/discord.js/src/util/APITypes.js) file